src/content/docs/iast/install.mdx

```diff
@@ -50,14 +50,38 @@ To install New Relic IAST:
             title={<><InlineCode>newrelic.yml</InlineCode> config file</>}
 >
                 ```yml
-                host: staging-collector.newrelic.com
                 security:
                     enabled: true
-                agent:
+                    agent:
+                        enabled: true
+                ```
+        </Collapser>
+        <Collapser
+            id="config-file-example"
+            title={<><InlineCode>newrelic.yml</InlineCode> config file for EU</>}
```

Comment:
Shouldn't these just be inline code wrapped like this: ``textHere` `?

I think using html code tags might mess up the translation workflow

Reply:
I've seen this structure in our doc site with filenames. I'm going to ask the question to our eng team. Thanks!